dev/docs/supported-features.md

```diff
@@ -130,3 +130,12 @@ Supported features are marked with a grey checkbox
 - [x] TCPRoute - we are limited by Consul's ability to only route to a single TCP-based upstream, so TCP-based Gateway listeners only support a single TCPRoute with a single rule with a single backend; otherwise, either the TCPRoute is considered invalid or the Gateway status is set as having conflicting routes.
 - [ ] TLSRoute - TODO
 - [x] UDPRoute *not supported*
+- [x] ReferencePolicy - Routes currently require ReferencePolicies for all BackendRefs, and do not support partial acceptance if any BackendRefs are unpermitted
+  - [x] From
+    - [x] HTTPRoute
+    - [x] TCPRoute
+    - [ ] TLSRoute
+    - [ ] UDPRoute
```

Comment:
```suggestion
    - [ ] UDPRoute
    - [ ] Gateway
```

since we include Secret as a future `To` target, figured we should add Gateway as a `From`

Reply:
👍 I just wasn't sure if that was the intended `From` target, hadn't seen it mentioned explicitly anywhere upstream yet.